### ISSUE TITLE
libsuperfunpack.so: undefined symbol: pcre_compile

### ISSUE THREAD

**Issue by rvernica** [Paradigm4/extra-scidb-libs#19]
After installing `extra-scidb-libs` using the `install.sh` script and loading the `superfunpack` plugin, using `rsub` in a query, will bring SciDB down with:
```
symbol lookup error: /opt/scidb/18.1/lib/scidb/plugins/libsuperfunpack.so: undefined symbol: pcre_compile
```
Compiling `superfunpack` locally solves the problem. No extra packages need to be installed. This happened on Ubuntu `14.04`.

**Comment by rvernica** [comment]
Here is how to reproduce this:
```bash
> docker run -ti --rm rvernica/scidb:18.1-trusty bash
 * Starting OpenBSD Secure Shell server sshd                             [ OK ] 
 * Starting PostgreSQL 9.3 database server                               [ OK ] 
...

# wget -O- https://paradigm4.github.io/extra-scidb-libs/install.sh | sh
--2018-09-21 02:44:11--  https://paradigm4.github.io/extra-scidb-libs/install.sh
Resolving paradigm4.github.io (paradigm4.github.io)... 185.199.111.153, 185.199.108.153, 185.199.109.153, ...
Connecting to paradigm4.github.io (paradigm4.github.io)|185.199.111.153|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 3992 (3.9K) [application/x-sh]
Saving to: 'STDOUT'

100%[======================================>] 3,992       --.-K/s   in 0.001s  
...

# iquery -a
AFL% load_library('superfunpack');
Query was executed successfully

AFL% apply(build(<s:string>[i=0:0,1,0],'{0}[(\'Paul Brown is a serious serious man.\')]',true), r, rsub(s,'s/serious/silly/'));
SystemException in file: src/network/BaseConnection.h function: receive line: 171
Error id: scidb::SCIDB_SE_NETWORK::SCIDB_LE_CANT_SEND_RECEIVE
Error description: Network error. Cannot receive network message: Read failed: End of file (asio.misc:2).  An instance may be offline..

# tail /opt/scidb/18.1/DB-scidb/0/0/scidb-stderr.log 
2018-09-21 02:44:05 (ppid=161): Started.
/opt/scidb/18.1/DB-scidb/0/0/SciDB-0-0-scidb: symbol lookup error: /opt/scidb/18.1/lib/scidb/plugins/libsuperfunpack.so: undefined symbol: pcre_compile
2018-09-21 02:51:34 (ppid=161): SciDB child (pid=168) exited with status = 127
```

**Comment by rvernica** [comment]
This is due to `-lpcre` flag and has been addressed in [superfunpack](https://github.com/Paradigm4/superfunpack/commit/904e0d88e8e27164c3cd4875869e2dced2a16717)

**Comment by rvernica** [comment]
Fixed in [v18.1.6](https://github.com/Paradigm4/extra-scidb-libs/releases/tag/v18.1.6)
